components/menus/planning.ts

```diff
@@ -357,94 +467,7 @@ export async function planningView(
-                          },
-                          {
-                              SlotName: "disguise",
-                              SlotId: "3",
-                              Recommended: {
-                                  item: typedInv.find(
-                                      (item) => item.Unlockable.Id === suit,
-                                  ),
-                                  type: "disguise",
-                              },
-                          },
-                          {
-                              SlotName: "gear",
-                              SlotId: "4",
-                              Recommended: {
-                                  item:
-                                      contractData.Peacock?.noGear === true
-                                          ? null
-                                          : typedInv.find(
-                                                (item) =>
-                                                    item.Unlockable.Id ===
-                                                    tool1,
-                                            ),
-                                  type: "gear",
-                              },
-                          },
-                          {
-                              SlotName: "gear",
-                              SlotId: "5",
-                              Recommended: {
-                                  item:
-                                      contractData.Peacock?.noGear === true
-                                          ? null
-                                          : typedInv.find(
-                                                (item) =>
-                                                    item.Unlockable.Id ===
-                                                    tool2,
-                                            ),
-                                  type: "gear",
-                              },
-                          },
-                          {
-                              SlotName: "stashpoint",
-                              SlotId: "6",
-                              Recommended: null,
-                          },
-                          briefcaseId && {
-                              SlotName: briefcaseProp,
-                              SlotId: briefcaseId,
-                              Recommended: {
-                                  item: {
-                                      ...i,
-                                      Properties: {},
-                                  },
-                                  type: i.Unlockable.Id,
-                                  owned: true,
-                              },
-                              IsContainer: true,
-                          },
-                      ].filter(Boolean),
+                contractData.Metadata.Type === "sniper" ? null : loadoutSlots,
             LimitedLoadoutUnlockLevel: 0, // Hokkaido
```

Comment:
Consider editing the `LimitedLoadoutUnlockLevel` to be 20 for Hokkaido and 2 for Miami. It will be reflected in the UI.

Reply:
@moonysolari Is this to replace the filtering? Or to do together with it?

Reply:
It has nothing to do with the filtering I think. On line 471 just set `LimitedLoadoutUnlockLevel` to 20 for Hokkaido, 2 for Miami, and 0 for everything else?

Reply:
Yeah sorry, I though this was to lock the loadouts based on the level, I realize now its to display the required mastery on the loadout slot :) Should be addressed now 👍